packages/@icon-magic/distribute/src/distribute-svg.ts

```diff
@@ -187,6 +187,10 @@ function getAssetsToDistribute(icon: Icon, type: FlavorType, colorScheme: string
     assetsToDistribute = assetsToDistribute.filter(asset => {
       return asset.name.match(/-with-image/) ? true : false;
     });
+  } else {
+    assetsToDistribute = assetsToDistribute.filter(asset => {
+      return asset.name.match(/-with-image/) ? true : false;
+    });
```

Comment:
Am I missing something but if you put an else statement here to the if statement above, and they are the same on the inside.. don't you just need to it without the if statement?  Isn't it going to happen no matter what?

Reply:
oh oops updated